sonarpedia.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules-metadata-path": ".\\sonaranalyzer-dotnet\\rspec\\cs",
+  "languages": [
+    "CSH"
```

Comment:
Shouldn't that be pointing to VB.NET?

Reply:
No this change is for C#. VB one is on VB repo.